{"name": "test_fail7", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_fail7():\n>       assert False\nE       assert False\n\ntests/simple/test_fail.py:20: AssertionError"}, "start": 1759909735339, "stop": 1759909735339, "uuid": "592ec01c-39a2-415e-ba8c-055dea55676c", "historyId": "e5476871c2923afbd50a8191a77cef77", "testCaseId": "e5476871c2923afbd50a8191a77cef77", "fullName": "tests.simple.test_fail#test_fail7", "labels": [{"name": "parentSuite", "value": "tests.simple"}, {"name": "suite", "value": "test_fail"}, {"name": "host", "value": "MacBook-Air.local"}, {"name": "thread", "value": "36329-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.simple.test_fail"}], "titlePath": ["tests", "simple", "test_fail.py"]}